app/uk/gov/hmrc/fileupload/controllers/EnvelopeController.scala

```diff
@@ -27,27 +29,41 @@ import uk.gov.hmrc.fileupload.utils.JsonUtils.jsonBodyParser
 import uk.gov.hmrc.fileupload.write.envelope._
 import uk.gov.hmrc.fileupload.write.infrastructure._
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId, _}
-import uk.gov.hmrc.play.microservice.controller.BaseController
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 
+
+package object envelope {
+
+  import play.api.libs.concurrent.Execution.Implicits._
+
+  object EnvelopeController extends EnvelopeController(
+    withBasicAuth = MicroserviceGlobal.withBasicAuth,
```

Comment:
I was going to ask about package objects as well but you already answered that this is for importing execution context. Some package objects have a name of `impl` whilst others don't. Was this a conscious decision?